docs/src/content/docs/fr/manual-setup.mdx

```diff
@@ -0,0 +1,129 @@
+
+export default defineConfig({
+	integrations: [
+		starlight({
+      title: 'Mon ravissant site de documentation'
+		}),
+	],
+});
+```
+
+Retrouvez toutes les options disponibles dans la [référence de configuration Starlight](/fr/reference/configuration/).
+
+### Configurer les collections de contenu
+
+Starlight s'appuie sur les [collections de contenu d'Astro](https://docs.astro.build/fr/guides/content-collections/), qui sont configurées dans le fichier `src/content/config.ts`.
+
+Créez ou mettez à jour le fichier de configuration du contenu, en ajoutant une collection `docs` qui utilise le schéma `docsSchema` de Starlight :
+
+
+```js ins={3,6}
+// src/content/config.ts
+import { defineCollection } from 'astro:content';
+import { docsSchema } from '@astrojs/starlight/schema';
+
+export const collections = {
+	docs: defineCollection({ schema: docsSchema() }),
+};
+```
+
+### Ajouter du contenu
+
+Starlight est maintenant configuré et il est temps d'ajouter du contenu !
+
+Créez un repetoire `src/content/docs/` et commencez par ajouter un fichier `index.md`.
+Ce sera la page d'accueil de votre nouveau site :
+
+```md
+---
+# src/content/docs/index.md
+title: Ma documentation
+description: En savoir plus sur mon projet dans ce site de documentation construit avec Starlight.
+---
+
+Bienvenue dans mon projet !
+```
+
+Starlight utilise une stratégie de routage basée sur les fichiers, ce qui signifie que chaque fichier Markdown, MDX ou Markdoc dans `src/content/docs/` deviendra une page de votre site. Les métadonnées du frontmatter (les champs `title` et `description` dans l'exemple ci-dessus) peuvent modifier la façon dont chaque page est affichée.
+Retrouvez toutes les options disponibles dans la [référence du frontmatter](/fr/reference/frontmatter/).
+
+## Astuce pour les sites existants
+
+Si vous avez un projet d'Astro existant, vous pouvez utiliser Starlight pour ajouter rapidement une section de documentation à votre site.
+
+### Utiliser Starlight dans un sous-chemin
+
+Pour ajouter toutes les pages Starlight dans un sous-chemin, placez tout votre contenu de documentation dans un sous-répertoire de `src/content/docs/`.
+
+Par exemple, si les pages de Starlight doivent toutes commencer par `/guides/`, ajoutez votre contenu dans le répertoire `src/content/docs/guides/` :
+
+import FileTree from '../../../components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - **guides/**
+        - guide.md
+        - index.md
+  - pages/
+- astro.config.mjs
+
+</FileTree>
+
+À l'avenir, nous prévoyons de mieux supporter ce cas d'utilisation pour éviter le besoin du répertoire supplémentaire dans `src/content/docs/`.
+
+### Utiliser Starlight avec SSR
+
+À l'heure actuelle, Starlight ne supporte pas [le déploiement rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```

Comment:
Looks like the second word in @HiDeoo’s review was missed here:

```suggestion
À l'heure actuelle, Starlight ne supporte pas [le déploiement avec rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```

Reply:
oof, nice catch @delucis